examples/kepler-integration/src/factories/export-modal.js

```diff
@@ -0,0 +1,29 @@
+import {SaveExportDropdownFactory, withState} from 'kepler.gl/components';
+import {toggleHubbleExportModal} from '../actions';
+import {Icons} from 'kepler.gl/components';
+
+const CustomSaveExportDropdownFactory = (...deps) => {
+  const SaveExportDropdown = SaveExportDropdownFactory(...deps);
+  const defaultLoadingMethods = SaveExportDropdown.defaultProps.items;
+
+  const exportVideoModal = {
+    label: 'toolbar.exportVideoModal',
+    icon: Icons.Files, // TODO: video icon
+    key: 'video',
+    onClick: props => _ => props.toggleHubbleExportModal(true)
```

Comment:
From Raymond: Normally expects props from PanelHeaderFactory, but this is being used as part of Hubble. Otherwise will get filtered out. Reference https://github.com/keplergl/kepler.gl/blob/master/src/components/side-panel/panel-header.js#L139